ydb-cookbook/src/test/java/tech/ydb/examples/ExamplesTest.java

```diff
@@ -51,7 +52,12 @@ public void testReadTable() {
 
     @Test
     public void testPagination() {
-        Assertions.assertEquals(0, PaginationApp.test(args(), "Pagination test"));
+        Assertions.assertEquals(0, PaginationApp.test(args()), "Pagination test");
```

Comment:
But you removed second argument from PaginationApp.test. Should it really be here?
Same question for testReadRows()

Reply:
This string literal is a custom message for assertion error. There was a typo when this literal was passed to PaginationApp.test(..) where it was ignored. All others methods like `testBulkUpsert`, `testBatchUpload`, `testReadRows` don't have this error.

Reply:
My bad, I just missed the parenthesis now moved to the left